crates/wast/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wast"
-version = "27.0.0"
+version = "27.1.0"
```

Comment:
Ah unfortunately the changes in this crate are almost always public-facing, so this'll need to go to 28.0.0